### PR TITLE
Update with new tests to be done during deployments

### PR DESCRIPTION
**Pull request by chartjes**

### CODE REVIEW

**Comment by chartjes** [comment]
@rpappalax r?

File: manifest.ini [hunk 1]
```diff
@@ -3,26 +3,26 @@ server_url = https://updates-autopush.stage.mozaws.net
 push_server = wss://autopush.stage.mozaws.net
 router = stage.autopush.routerv2
 storage = stage.autopush.storage
-bugzilla_api = https://bugzilla-dev.allizom.org
-bugzilla_user = chartjes@mozilla.com
-bugzilla_api_key = oelpF8y4nHYuYf3xOxll4XQXTJtPAkIwrs1BmHvO 
+bugzilla_api =
```
Comment (rpappalax) [inline manifest.ini]: don't you still require these for the test? if so, you could invoke them as ENV vars like so: https://github.com/mozilla-services/autograph-loadtests/blob/master/loadtest.py#L21-L26
then you can just set those values locally in your bash_profile and later, on Jenkins, we'll just add them as ENV vars as well

Comment (rpappalax) [reply manifest.ini]: also, if these have been removed from the project, there's prob no reason to keep these empty keys here any longer

File: manifest.ini [hunk 2]
```diff
@@ -3,26 +3,26 @@ server_url = https://updates-autopush.stage.mozaws.net
...
 
 [production]
 server_url = https://updates.push.services.mozilla.com
 push_server = wss://push.services.mozilla.com
 router = prod.autopush.routerv2
 storage = prod.autopush.storage
-bugzilla_api = https://bugzilla-dev.allizom.org
-bugzilla_user = chartjes@mozilla.com
-bugzilla_api_key = oelpF8y4nHYuYf3xOxll4XQXTJtPAkIwrs1BmHvO
+bugzilla_api =
```
Comment (rpappalax) [inline manifest.ini]: (see above)

File: manifest.ini [hunk 3]
```diff
@@ -3,26 +3,26 @@ server_url = https://updates-autopush.stage.mozaws.net
...
 
 [dev]
 server_url = https://autopush-rust.dev.mozaws.net
 push_server = wss://autopush-rust.dev.mozaws.net
 router = dev.autopush.routerv2
 storage = dev.autopush.storage
-bugzilla_api = https://bugzilla-dev.allizom.org
-bugzilla_user = chartjes@mozilla.com
-bugzilla_api_key = oelpF8y4nHYuYf3xOxll4XQXTJtPAkIwrs1BmHvO
+bugzilla_api =
```
Comment (rpappalax) [inline manifest.ini]: (same here)

File: manifest.ini [hunk 4]
```diff
@@ -3,26 +3,26 @@ server_url = https://updates-autopush.stage.mozaws.net
...
-bugzilla_api = https://bugzilla-dev.allizom.org
-bugzilla_user = chartjes@mozilla.com
-bugzilla_api_key = oelpF8y4nHYuYf3xOxll4XQXTJtPAkIwrs1BmHvO 
+bugzilla_api =
+bugzilla_user =
+bugzilla_api_key =
 
 
 [production]
 server_url = https://updates.push.services.mozilla.com
```
Comment (rpappalax) [inline manifest.ini]: we eventually want to pass all of these URLs dynamically from ops Jenkins to our Jenkins so these will also likely be passed in as ENV vars.  We could always do this later when we wire these tests up w/ CI

File: manifest.ini (hunk 1 above)
Comment (chartjes) [reply manifest.ini]: I am going to be using them but just not yet -- still experimenting with changes to the BugZilla posting library I will be using. I wanted to keep them in as a place-holder

**Comment by rpappalax** [comment]
@chartjes sounds like a work-in-progress, so take my comments with a grain of salt.  looks like removing import should clear your test in travis.  :+1:

File: manifest.ini (hunk 4 above)
Comment (chartjes) [reply manifest.ini]: I don't pass them in dynamically for Kinto -- we pass in the stack we are testing (dist, settings, webextensions) and then target testing environment (stage, production)

**Pull request merged**